tests/breakpoint.spec.scss

```
@@ -156,3 +156,113 @@
     }
   }
 }
+
+@include describe('auro_grid-breakpoint--xl()') {
+  @include it('should return content within pre-defined media query') {
+    @include assert {
+      @include output {
+        .content {
+          @include auro_grid-breakpoint--xl {
+            content: "Grid media query for xl";
+          }
+        }
+      }
+
+      @include expect {
+        @media screen and (min-width: 1232px) {
```

```suggestion
        @media screen and (min-width: var(--auro-grid-breakpoint-xl)) {
```

See previous comment about replacing Sass variables with CSS custom properties.